sql/sql_acl.cc

```diff
@@ -12273,7 +12273,7 @@ static bool find_mpvio_user(MPVIO_EXT *mpvio)
     DBUG_ASSERT(my_strcasecmp(system_charset_info, mpvio->acl_user->plugin.str,
                               old_password_plugin_name.str));
     my_error(ER_NOT_SUPPORTED_AUTH_MODE, MYF(0));
-    general_log_print(mpvio->thd, COM_CONNECT,
+    general_log_print(mpvio->thd, COM_CONNECT, "%s",
                       ER_THD(mpvio->thd, ER_NOT_SUPPORTED_AUTH_MODE));
```

Comment:
Thinking more on this: ER_NOT_SUPPORTED_AUTH_MODE is a format string itself. So original code was just fine.